### ISSUE TITLE
About training loss

### ISSUE THREAD

**Issue by UtopiaHu** [VITA-Group/EnlightenGAN#82]
Hi, 
Thank you for your work. I have a question. Can we get some information from the loss curve during training? What is the normal loss curve like during training? How can I tell the training is not convergent according to the loss curve?

**Comment by yueruchen** [comment]
The VGG loss is increasing while the adversarial loss is decreasing during training stage. There is no typical signal to show whether a GAN model converges.

**Comment by UtopiaHu** [comment]
So, 'The VGG loss is increasing while the adversarial loss is decreasing during training stage. ' means the model is not convergent ?

**Comment by yueruchen** [comment]
on this case and dataset it is not
